### PR TITLE
Add new environment variable to DevHub jobs

### PR DESCRIPTION
**Pull request by jestapinski**
Ahoy!

Our team is introducing a new DevHub service and wish to do so using an environment variable. The goal of this PR is to add the environment variable to all DevHub jobs. If I missed something please let me know!

Thanks in advance,
Jordan

### CODE REVIEW

File: makefiles/Makefile.devhub-content-integration [hunk 1]
```diff
@@ -57,4 +60,4 @@ next-gen-html-publish:
 
 next-gen-stage: ## Host online for review
 	mut-publish public ${STAGING_BUCKET} --prefix="${COMMIT_HASH}/${MUT_PREFIX}" --stage ${ARGS};
-	echo "Hosted at ${STAGING_URL}/${COMMIT_HASH}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
\ No newline at end of file
+	echo "Hosted at ${STAGING_URL}/${COMMIT_HASH}/${MUT_PREFIX}/${USER}/${GIT_BRANCH}/";
```
Comment (madelinezec) [inline makefiles/Makefile.devhub-content-integration:63]: could we remove the newline added to 63?

Comment (jestapinski) [reply makefiles/Makefile.devhub-content-integration:63]: For sure! I think my editor automatically added this.

**Review by madelinezec — CHANGES_REQUESTED** [review]
Looks great! we have to be annoyingly nit-picky bc makefiles are annoyingly nit-picky. 

Also, if you would like the ability to test makefile changes before they get pushed to production, let me know and I can arrange that

**Comment by jestapinski** [comment]
Thanks Maddie! It might be useful for us to test these jobs, so sure I will certainly take that!

**Comment by jestapinski** [comment]
https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?jobId=5ffe1ca0c5bd3658da8000a0 (intgegration job)
and
https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?jobId=5ffe1cdec5bd3658da800ebe (devhub-content job)

**Review by madelinezec — APPROVED** [review]
LGTM

**Pull request merged**